### DISCUSSION TITLE
pre-trained model on MIMIC-III

### DISCUSSION THREAD

**Discussion by sherry6247** [MIT-LCP/mimic-code#1384] [MIMIC-III] [answered]
Hi all,
I have pre-trained a model based on the MIMIC-III dataset, using structured and unstructured data. May I put this pre-trained model  (including the pre-trained model parameters and the vocabulary of codes and notes ) on the public code repositories?

**Comment by tompollard** [comment, marked as answer]
Hi @sherry6247, thanks for checking. It sounds like your model has potential to leak sensitive information. We support sharing of this kind of model through PhysioNet (see: https://physionet.org/about/publish/#author_guidelines), allowing you to release it under the same access restrictions as the source data. Would this work for you?

**Reply by sherry6247** [reply]
Hi @tompollard , thanks for this suggestion, however, when I try to submit a model on physionet website, it also shows me that "Outstanding storage request" How can I fix this request to success submit my project on physionet?

**Reply by tompollard** [reply]
All storage requests are reviewed by the PhysioNet team during standard working hours (9-5, Mon-Fri). It looks like you requested more storage over the weekend, on a Sunday. We have now approved your request, and you are able to submit,.

**Reply by sherry6247** [reply]
Oops, I ignored this request. Thank you for your reply. I have completed my submission.
Best wishes.
